Formula/augustus.rb

```diff
@@ -40,7 +40,7 @@ def install
     # to upstream in 2016 (see https://github.com/nextgenusfs/funannotate/issues/3).
     # See also https://github.com/Gaius-Augustus/Augustus/issues/64
     cd "src" do
-      with_env("HOMEBREW_CC" => Formula["gcc"].opt_bin/"gcc-#{Formula["gcc"].version_suffix}") do
+      with_env("HOMEBREW_CC" => Formula["gcc"].opt_bin/"gcc-#{Formula["gcc"].linked_version.to_s.slice(/\d+/)}") do
```

Comment:
I feel like we could do something better than this; maybe an extension to strings like `Formula["gcc"].linked_version.version_suffix`?

Reply:
Good idea. Merging this for now.